TowerForge/TowerForge/LevelManager/Components/BaseComponents/LabelComponent.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  LabelComponent.swift
+//  TowerForge
+//
+//  Created by Vanessa Mae on 23/03/24.
+//
+
+import Foundation
+import SpriteKit
+
+class LabelComponent: TFComponent {
+    var text: String
+    init(text: String) {
+        self.text = text
+        super.init()
+    }
+    override func didAddToEntity(_ entity: TFEntity) {
+        self.entity = entity
+        guard let spriteComponent = entity.component(ofType: SpriteComponent.self) else {
+            return
+        }
+        let label = SKLabelNode(text: self.text)
+        label.fontName = "HelveticaNeue-Bold"
+        label.fontSize = 60.0
+        label.fontColor = .white
+        label.horizontalAlignmentMode = .center
+        label.verticalAlignmentMode = .center
+        label.position = CGPoint(x: spriteComponent.node.width, y: 0)
+        label.name = "point"
+        spriteComponent.node.addChild(label)
```

Comment:
Would it be better to shift this to Renderer so that all view logic is separate from functional logic and there is proper separation of concerns?

Reply:
moved to renderer